WordPress/Classes/ViewRelated/Comments/EditCommentTableViewController.swift

```diff
@@ -13,6 +18,14 @@ class EditCommentTableViewController: UITableViewController {
 
     // MARK: - Init
 
+    @objc convenience init(comment: Comment) {
+        self.init()
+        authorName = comment.author
+        commentContent = comment.contentForEdit()
+        authorWebAddress = comment.author_url
+        authorEmailAddress = comment.author_email
```

Comment:
Comment data is stored separately here on purpose (as opposed to just storing the Comment). Notifications doesn't use a `Comment` object, so it will have to provide the data separately. Having each stored separately here will accommodate for that.